packages/components/src/components/UtilityCard/UtilityCard.tsx

```diff
@@ -77,7 +77,7 @@ export const UtilityCard: React.FC<UtilityCardProps> = ({
         padding={{ _: "d4", md: "d6" }}
         w={{ _: "56px", md: "56px" }}
       >
-        <Box.img alt="" aria-hidden h="24px" src={imageSrc} w="24px" />
+        <Box.div aria-hidden="true">{emoji}</Box.div>
```

Comment:
I would change it from `Box.div` to `Box.span` because the emoji acts like a simple text.